crates/lib/src/qpu/quilc/isa/qubit.rs

```diff
@@ -239,28 +239,28 @@ mod describe_rx_gates {
             Operator::Gate {
                 arguments: vec![Argument::Int(1)],
                 duration: 50.0,
-                fidelity: 0.996_832_609_176_635_7,
+                fidelity: 0.996_832_6,
```

Comment:
@notmgsk The original characteristic from line 221 has this value exactly, the extra digits on the pre-change number I figure is due to `i32 -> i64` conversion which this MR gets rid of, but wanted to double check with you that I wasn't missing anything

```
            Characteristic {
                name: "fRB".to_string(),
                value: 0.996_832_6,
                error: Some(0.000_100_896_78),
                node_ids: Some(vec![1]),
                timestamp: "1970-01-01T00:00:00+00:00".to_string(),
                parameter_values: None,
            },
```

Reply:
Assuming you mean `f32 -> f64` and so the loss of precision in parsing, then this seems good to me.